docu/sphinx/source/conf.py

```diff
@@ -109,11 +109,11 @@ def getDocRoot():
     'github_repo': 'igor-unit-testing-framework',
     'github_banner': True,
     'show_powered_by': False,
-    'page_width' : '90%'
+    'page_width' : '90%',
 }
 
 html_context = {
-    'full_revision' : "Version: {}".format(getRelease())
```

Comment:
version and revision are already available. No need to do String functions. I modified the template to get the "Version:" Label.